test/e2e/density.go

```diff
@@ -250,6 +250,30 @@ var _ = Describe("Density [Skipped]", func() {
 			stop := make(chan struct{})
 			go controller.Run(stop)
 
+			// Create a listener for api updates
+			updateCount := 0
```

Comment:
As we discussed offline, `updateCount` is accessed by multiple goroutines and will need to be fixed.